functions_lab.js

```diff
@@ -0,0 +1,276 @@
+/*Write a function that returns the largest of two numbers. If they are equal, return the second one.
```

Comment:
Good demonstration of functions and scope. No major errors or feedback for this assignment. Keep up the good work!